.github/workflows/lint.yml

```
@@ -26,7 +26,7 @@ jobs:
       - name: Black code style
         run: black larq_compute_engine --check --target-version py36 --exclude 'build/|buck-out/|dzzist/|_build/|\.git/|\.hg/|\.mypy_cache/|\.tox/|\.venv/|larq/snapshots/'
       - name: clang-format lint
-        uses: DoozyX/clang-format-lint-action@v0.11
+        uses: DoozyX/clang-format-lint-action@v0.12
         with:
           clangFormatVersion: 11
```

Do we also want to update the clang version for formatting, or should we wait with that?
```suggestion
           clangFormatVersion: 12
```